src/App.tsx

```diff
@@ -1,23 +1,17 @@
 import React from 'react';
-import logo from './logo.svg';
+//import logo from './logo.svg';
```

Comment:
we can probably just delete this line of code instead of commenting it out.

Reply:
Deleted